### DISCUSSION TITLE
My 3.5" Theme collection

### DISCUSSION THREAD

**Discussion by gonztok** [mathoudebine/turing-smart-screen-python#417] [Themes]
Just wanted to share my Themes for the 3.5" screen:

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/68f028f2-2b32-4519-8eb8-0a9e80f70a0a)

![preview (1)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/5a749ffe-2378-42f6-be92-4d234fde7a78)

![preview (2)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/982b20d7-63a5-4162-98d7-edd86d22b772)

![preview (3)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/662293be-8c02-40b5-aea1-32bfd9384470)

![preview (4)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/f092b3c2-291f-4023-bc14-5b3088bab161)

![preview (5)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/abcdb9f9-3d51-4688-a67b-4ba4d4730b5e)

![preview (6)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/aec8588a-5d79-4eb7-9891-50c6e2d1fe4e)

![preview (7)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/4633fb8a-9967-4e9c-97ee-07012fea792e)

![preview (8)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/cb0971c1-3c7a-4da7-be36-4b7dfa399792)

![preview (9)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/0e0e0dd6-ab15-4c38-b441-bbe51d129189)

![preview (10)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/ca14101d-f1cf-447c-b647-5a58f213bae0)

![preview (11)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/c2e2da67-a324-4729-b5bf-3033413fb345)

![preview (12)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/c6847b51-fc60-4d41-b2d3-07541153a8e3)

![preview (13)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/59ca5c7a-3ce2-48d2-850b-58c84b94944b)

![preview (14)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/8082709b-5d04-4c12-ab16-09b930da9afa)

![preview (15)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/52a0c139-cfd7-4ff1-bbb9-3c205cec52a5)

![preview (16)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/61131471-4065-4592-ad6f-3000fc9e19f4)

![preview (17)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/b06fcbe1-4eac-4419-aecb-eb167e081399)

![preview (18)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/da92a8c7-4d8a-4b38-a289-af86fad73210)

![preview (19)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/8a338f6e-05e9-416f-89f6-42f4b156d191)

![preview (20)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/02629da0-d521-4b63-b253-73df87eee72a)

![preview (21)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/aeb15ed9-b6a7-4fa1-a5fc-ed9fc3266414)

![preview (22)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/f57fbddc-8e3c-408c-ab8f-7fb7e2cafb0e)

![preview (23)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/ca8534d9-cebf-4ed1-8d34-25ef3ef64ef5)

![preview (24)](https://github.com/mathoudebine/turing-smart-screen-python/assets/5973936/4ea4068d-0132-44a0-85ba-a86bb34db194)

[Smart Monitor Themes.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/13691353/Smart.Monitor.Themes.zip)

**Comment by DJRoby19** [comment]
very nice compliments..

**Comment by sysyphus** [comment]
Thx !!!

**Comment by MackenzyARG** [comment]
Hi, my files are .data, that's how they came up on my screen. How can I put the ones I downloaded here? Thanks.
